tinygrad/multi.py

```diff
@@ -110,8 +110,8 @@ def alu(self, op:Union[MetaOps, UnaryOps, BinaryOps, TernaryOps], *in_srcs:Multi
     assert any(new_real), "output contains no real lb"
     for mlb in msrcs:
       if (mlb.axis == axis and (mlb.axis is None or mlb.bounds == bounds)) or not_all_real: srcs.append(mlb.lbs)
-      elif mlb.axis is None and axis is not None: srcs.append(to_sharded(mlb.lbs, axis, bounds))
-      else: srcs.append(to_sharded([mlb.copy_to_device(lb.device) for lb in mlb.lbs], axis, bounds))
+      elif mlb.axis is None and axis is not None and bounds is not None: srcs.append(to_sharded(mlb.lbs, axis, bounds))
+      elif axis is not None and bounds is not None: srcs.append(to_sharded([mlb.copy_to_device(lb.device) for lb in mlb.lbs], axis, bounds))
```

Comment:
Huh? This change behavior?

Reply:
`to_sharded` doesn't accept None attributes so I don't think this changes behavior

Reply:
Doesn't accept?!? What do you mean.

Reply:
Sorry, I meant to say that it will throw an error